dags/inference/configs/jetstream_benchmark_serving_gce_config.py

```diff
@@ -84,7 +84,7 @@ def get_config(
       "quantization": f"{model_configs['quantization']}",
       "quantize_kvcache": f"{model_configs['quantize_kvcache']}",
       "weight_quant_dtype": f"{model_configs['quantization'] if model_configs['quantization'] else 'bf16'}",
-      "kvcache_quant_dtype": f"{'int8' if model_configs['quantize_kvcache'] else 'bf16'}",
+      "kvcache_quant_dtype": f"{'int8' if model_configs['quantize_kvcache'].lower() == 'true' else 'bf16'}",
```

Comment:
I think right now we also support int4 for kvcache_quant_dtype

Reply:
Got it, thanks for the pointer. I updated this variable to `kv_quant_dtype`. PTAL